shakenfist/scheduler.py

```diff
@@ -209,6 +209,15 @@ def place_instance(self, instance, network, candidates=None):
             db.add_event('instance', instance.db_entry['uuid'],
                          'schedule', 'Have most matching images', None, str(candidates))
 
+            # Avoid allocating to network node if possible
+            net_node = db.get_network_node()
+            if len(candidates) > 1 and net_node['fqdn'] in candidates:
+                candidates.remove(net_node['fqdn'])
+                LOG.info('Scheduling %s, %s are non-network nodes'
+                            % (instance, candidates))
+                db.add_event('instance', instance.db_entry['uuid'],
+                             'schedule', ' are non-network nodes', None, str(candidates))
```

Comment:
Sorry to be a pain, but I think that should be "Are non-network nodes" to be consistent with the others here?

Reply:
Absolutely.